packages/@ember/-internals/deprecations/index.ts

```diff
@@ -121,7 +121,7 @@ export const DEPRECATIONS = {
   }),
   DEPRECATE_COMPONENT_TEMPLATE_RESOLVING: deprecation({
     id: 'component-template-resolving',
-    url: 'https://deprecations.emberjs.com/id/component-template-resolving',
+    url: 'https://deprecations.emberjs.com/v5.x/#toc_deprecate-component-template-resolving',
```

Comment:
Should be: https://deprecations.emberjs.com/v5.x/#toc_deprecate-component-template-resolving